src/pages/beheer/instellingen/index.tsx

```diff
@@ -7,12 +7,56 @@ import { useState } from 'react';
 import { withAuth } from '@auth/wrappers/with-auth';
 import { CPAdminLayout } from '@cp/layouts';
 import { withI18n } from '@i18n/wrappers';
+import { CardImage, Icon } from '@shared/components';
 import { RichTextEditor } from '@shared/components/RichTextEditor';
 import { createPageTitle } from '@shared/utils';
 
 const CPSettingsPage: NextPage = () => {
+	/**
+	 * Hooks
+	 */
 	const { t } = useTranslation();
-	const [editorState, setEditorState] = useState<RichEditorState>();
+
+	/**
+	 * Form state
+	 */
+
+	// Leeszaal
+	const [savedColor, setSavedColor] = useState<string>('#00857d'); // Save unedited state
+	const [newColor, setNewColor] = useState<string>('#00857d');
+
+	// Wachtzaal
+	const [savedWachtzaalState, setSavedWachtzaalState] = useState<RichEditorState>(); // Save unedited state
+	const [newWachtzaalState, setNewWachtzaalState] = useState<RichEditorState>();
+
+	// Aanvraag
+	const [savedAanvraagState, setSavedAanvraagState] = useState<RichEditorState>(); // Save unedited state
+	const [newAanvraagState, setNewAanvraagState] = useState<RichEditorState>();
```

Comment:
`yup` niet aan de praat gekregen voor deze pagina?

Reply:
`yup` is niet overal evident hier, maar komt in volgende pr.